gnuradio-runtime/include/gnuradio/logger.h

```diff
@@ -35,6 +35,7 @@ using logger_ptr = std::shared_ptr<void>;
 #include <gnuradio/api.h>
 #include <spdlog/common.h>
 #include <spdlog/fmt/fmt.h>
+#include <spdlog/fmt/ostr.h>
```

Comment:
I had to add this to allow PMTs to be logged without adding explicitly converting them to strings.

This will be useful for other modules, as there are other places where PMTs and other custom types are presently logged using Boost.Format.

Docs: https://github.com/gabime/spdlog#user-defined-types